inluxclient/query_test.go

```diff
@@ -0,0 +1,1083 @@
+			[][]interface{}{
+				{
+					"_result",
+					int64(0),
+					mustParseTime("2020-02-17T22:19:49.747562847Z"),
+					mustParseTime("2020-02-18T22:19:49.747562847Z"),
+					mustParseTime("2020-02-18T10:34:08.135814545Z"),
+					1.4,
+					"f",
+					"test",
+					"1",
+					"adsfasdf",
+				},
+				{
+					"_result",
+					int64(0),
+					mustParseTime("2020-02-17T22:19:49.747562847Z"),
+					mustParseTime("2020-02-18T22:19:49.747562847Z"),
+					mustParseTime("2020-02-18T22:08:44.850214724Z"),
+					6.6,
+					"f",
+					"test",
+					"1",
+					"adsfasdf",
+				},
+			},
+		},
+	}
+
+	verifyTables(t, csvTable, tables)
+}
+
+func TestQueryResultMultiTables(t *testing.T) {
+	csvTableMultiStructure := `#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,double,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true
+#default,_result,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,a,b
+,,0,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T10:34:08.135814545Z,1.4,f,test,1,adsfasdf
+,,0,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.850214724Z,6.6,f,test,1,adsfasdf
+
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,long,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true
+#default,_result,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,a,b
+,,1,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T10:34:08.135814545Z,4,i,test,1,adsfasdf
+,,1,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.850214724Z,-1,i,test,1,adsfasdf
+
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,boolean,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true
+#default,_result,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,a,b
+,,2,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.62797864Z,false,f,test,0,adsfasdf
+,,2,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.969100374Z,true,f,test,0,adsfasdf
+
+d
+#datatype,string,long,dateTime:RFC3339Nano,dateTime:RFC3339Nano,dateTime:RFC3339Nano,unsignedLong,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true
+#default,_result,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,a,b
+,,3,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.62797864Z,0,i,test,0,adsfasdf
+,,3,2020-02-17T22:19:49.747562847Z,2020-02-18T22:19:49.747562847Z,2020-02-18T22:08:44.969100374Z,2,i,test,0,adsfasdf
+
+`
+	tablesMultiStructure := []expectedTable{
+		{ // Table 1
+			[]influxclient.TableColumn{
+				{DataType: "string", DefaultValue: "_result", Name: "result", IsGroup: false},
+				{DataType: "long", DefaultValue: "", Name: "table", IsGroup: false},
```

Comment:
You can omit the empty `DefaultValue` fields here.

Reply:
Sure, but with it, there is explicitly shown I want to verify the empty string for the field.